### PR TITLE
Feature/pool 925 start finalizing production code test on

### PR DESCRIPTION
**Pull request by aodhgan**
WIP Instructions in scripts/fork/governance/runAll.sh

### CODE REVIEW

**Comment by aodhgan** [comment]
@asselstine

File: package.json [hunk 1]
```diff
@@ -57,6 +57,7 @@
     "ethereum-waffle": "^3.0.2",
     "ethers": "^5.0.8",
     "ganache-cli": "^6.12.0",
+    "hardhat": "^2.0.8",
```
Comment (asselstine) [inline package.json:60]: ✂️ 

I don't think this is being used anywhere atm

File: scripts/fork/governance/disbursePoolTokens.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+}
+
+function green() {
+  console.log(chalk.green.call(chalk, ...arguments))
+}
+
+const { ethers, deployments, getNamedAccounts } = buidler
+
+
+const employeeAAddress = require("../../../../governance/deployments/fork/TreasuryVesterForEmployeeA.json").address
```
Comment (asselstine) [inline scripts/fork/governance/disbursePoolTokens.js:15]: Ooooh relative paths....that's not portable.

Should at least set an environment variable to configure the gov project root

**Review by asselstine — CHANGES_REQUESTED** [review]

**Comment by aodhgan** [comment]
Closing another PR will superceed

**Pull request closed without merging**